ops/charm.py

```diff
@@ -729,7 +729,7 @@ class SecretChangedEvent(SecretEvent):
     a new secret revision, and all applications or units that are tracking this
     secret will be notified via this event that a new revision is available.
 
-    Typically, you will want to fetch the new content by calling
+    Typically, fetch the new content by calling
```

Comment:
I think this is a bit terse/odd. Perhaps "Typically, the charm will fetch...". Similar in a few places below.

Reply:
It seemed to me that the original request wasn't just getting rid of you/your but where possible moving to imperative language. "Typically, fetch the ..." is imperative, whereas "Typically, the charm will fetch the ..." is more descriptive. No problem changing it as you describe, but just wanted to clarify this first :)

Reply:
I agree we can/should move to more imperative language where possible. Here, however, I think it's less clear with just "Typically, fetch..." it's unclear who/what is doing the fetching: ops, the charm, something else? So I think the clarity here is helpful.